content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -1236,17 +1236,37 @@ object.
 
 ### Avoiding deadlocks in self-hosted webhooks
 
-A webhook running inside the cluster might cause deadlocks for its own deployment if it is configured
-to intercept resources required to start its own pods.
+There are several ways that webhooks can cause deadlocks, where the cluster cannot make progress in
+scheduling pods:
 
-For example, a mutating admission webhook is configured to admit `CREATE` pod requests only if a certain label is set in the
-pod (e.g. `"env": "prod"`). The webhook server runs in a deployment which doesn't set the `"env"` label.
-When a node that runs the webhook server pods
-becomes unhealthy, the webhook deployment will try to reschedule the pods to another node. However the requests will
-get rejected by the existing webhook server since the `"env"` label is unset, and the migration cannot happen.
+* A webhook running inside the cluster might cause deadlocks for its own deployment if it is configured
+  to intercept resources required to start its own pods.
 
-It is recommended to exclude the namespace where your webhook is running with a
-[namespaceSelector](#matching-requests-namespaceselector).
+  For example, a mutating admission webhook is configured to admit `CREATE` pod requests only if a certain label is set in the
+  pod (e.g. `"env": "prod"`). The webhook server runs in a deployment which doesn't set the `"env"` label.
+  When a node that runs the webhook server pods
+  becomes unhealthy, the webhook deployment will try to reschedule the pods to another node. However the requests will
+  get rejected by the existing webhook server since the `"env"` label is unset, and the migration cannot happen.
+
+  It is recommended to exclude the namespace where your webhook is running with a
+  [namespaceSelector](#matching-requests-namespaceselector).
```

Comment:
Let's mention ValidatingAdmissionPolicies as a way to provide (at least) some protection without having the admission webhook protect itself.

Reply:
Added a few lines down.